Tableau/Tableau_Desktop.download.recipe.yaml

```diff
@@ -1,19 +1,30 @@
-Description: Downloads latest Tableau Desktop disk image.
+Description: Downloads latest Tableau Desktop disk image for Intel or ARM architecture.
 Identifier: com.github.smithjw-actions.download.Tableau_Desktop
 MinimumVersion: '2.3'
 
 Input:
   NAME: Tableau Desktop
   SOFTWARE_TITLE: Tableau_Desktop
+  ARCHITECTURE: arm64
   BYPASS_STOP_PROCESSING_IF_DOWNLOAD_UNCHANGED: null
   DOWNLOAD_MISSING_FILE: null
+  INTEL_URL: https://www.tableau.com/downloads/desktop/mac
+  ARM_URL: https://www.tableau.com/downloads/desktop/reg-mac-arm64
```

Comment:
There's no need to have these URLs in the Inputs section, instead include them in two distinct `URLDownloaderPython` steps. We do this to make one package that will deploy to all devices